src/menu_display.c

```diff
@@ -0,0 +1,89 @@
+#include "cards_management.h"
+#include "game.h"
+
+void startScreen()
+{
+    printf("***************************************************************\n");
```

Comment:
Can we have the main menu displayed here itself?

Reply:
Good point. Will move it.